mne/morph.py

```diff
@@ -770,76 +770,6 @@ def _compute_morph_sdr(mri_from, mri_to, niter_affine=(100, 100, 10),
 
 ###############################################################################
 # Morph for SourceEstimate |  VectorSourceEstimate
-@deprecated("This function is deprecated and will be removed in version 0.19. "
```

Comment:
whasn't this one supposed to stay until 0.19 ??

Reply:
Argh yes this was errantly removed

Reply:
maybe we can change the warning to 0.18. and remove it here :D